packages/app/components/feed/comment-button.tsx

```diff
@@ -50,8 +52,9 @@ export function CommentButton({ nft }: CommentButtonProps) {
       size="regular"
       tw="h-auto p-0"
       onPress={handleOnPress}
+      accentColor={textColors}
     >
-      <Message height={24} width={24} />{" "}
+      <Message height={24} width={24} color={iconColor} />{" "}
       {nft?.comment_count > 0 ? getRoundedCount(nft.comment_count) : ""}
```

Comment:
I am not sure if this is semantically correct, should we wrap the "content" into a `<Text>` element? This should work on web, but React Native likes to keep the "text" wrapped into a <Text> element.

Reply:
yeah, it's works on native/web, I very much agree with your suggestion!  I also like the `${text} ` or `&nbsp;` way of writing, I will modify it.
It's just that it was written like this before, I didn't change it, maybe we define a specification.